__tests__/components/FormCommentsSort.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file is part of the Digital Planning Register project.
+ *
+ * Digital Planning Register is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Digital Planning Register is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Digital Planning Register. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { FormCommentsSort } from "@/components/FormCommentsSort";
+import { SearchParams } from "@/types";
+
+export interface FormCommentsSortProps {
+  council: string;
+  reference: string;
+  type: "public" | "specialist";
```

Comment:
We can use `import { CommentType } from "@/types/odp-types/schemas/postSubmissionApplication/enums/CommentType";` here now too 😄